contracts/protocol/StorageUtils.sol

```diff
@@ -70,14 +69,26 @@ contract StorageUtils is IStorageUtils {
             "Parameters too long"
         );
         templateId = keccak256(abi.encodePacked(endpointId, parameters));
+        emit AnnouncedTemplate(templateId, endpointId, parameters);
+    }
+
+    /// @notice Stores template details
+    /// @param endpointId Endpoint ID (allowed to be `bytes32(0)`)
+    /// @param parameters Template parameters, encoded in Airnode ABI
+    /// @return templateId Template ID
+    function storeTemplate(bytes32 endpointId, bytes calldata parameters)
+        external
+        override
+        returns (bytes32 templateId)
+    {
+        templateId = announceTemplate(endpointId, parameters);
```

Comment:
Looks kind of weird that we are not emitting the event at the end of the function but I do like re-using the code by calling the announce function 🤷🏻‍♂️